FinSearchUnified/Helper/StaticHelper.php

```diff
@@ -373,4 +374,20 @@ public static function isVersionLowerThan($version)
 
         return version_compare($shopwareVersion, $version, '<');
     }
+
+    public static function getShopwareVersion()
+    {
+        $version = null;
+        if (Shopware()->Container()->has('shopware.release.version')) {
+            $version = Shopware()->Container()->get('shopware.release.version');
+        } elseif (defined('\Shopware::VERSION')) {
+            $version = Shopware::VERSION;
+        }
+
+        if (!$version || $version === '___VERSION___') {
+            $version = getenv('SHOPWARE_VERSION') ?: '5.6.7';
```

Comment:
This was here before, but I guess if Shopware returns `___VERSION___`, we would have a problem if `SHOPWARE_VERSION` also isn't set in the environment, because then we wouldn't have a version at all, which is bad. This doesn't mean that I am suggesting hard coding a random version is better than no version at all.

Reply:
Should the default be the latest version? Currently it would be 5.6.9

Reply:
Sure, we can do that. Done.